joomla/joomla-cms
# Batch copy fields

Pull Request for Issue #37555

### Summary of Changes
Batch copy field did not generate new name and label and you end up with invalid data as you have multiple fields with the same name (but diffeerent id)

### Testing Instructions
Create three fields - one, two three
Use batch copy to copy those fields in the saame group
Verify that three new fields have been created - one (2), two (2), three, (3)
Publish all three fields
Open an article for editing

### Actual result BEFORE applying this Pull Request
The fields tab only has 3 fields not the expected 6



### Expected result AFTER applying this Pull Request
fields tab has 6 fields


### Documentation Changes Required
none
